src/app/products/charges/create-charge/create-charge.component.html

```diff
@@ -140,19 +154,26 @@
                 Amount is <strong>required</strong>
               </mat-error>
               <mat-error *ngIf="chargeForm.controls.amount.hasError('pattern') && currencyDecimalPlaces !== undefined">
-                Amount must be <strong>a positive number with {{ currencyDecimalPlaces ? 'up to ' + currencyDecimalPlaces : 'no' }} decimal places</strong>
+                Amount must be <strong>a positive number with {{ currencyDecimalPlaces ? 'up to ' +
+                  currencyDecimalPlaces : 'no' }} decimal places</strong>
               </mat-error>
               <mat-error *ngIf="chargeForm.controls.amount.hasError('pattern') && currencyDecimalPlaces === undefined">
                 Amount must be <strong>a positive number</strong>
               </mat-error>
             </mat-form-field>
-
-            <mat-form-field fxFlex="48%">
+            <!--
+   * minCap and maxCap only allowed for loan and savings entitites
+   * In Loan case: Only for the "charge calculation type" set as "%amount", "% loan amount+interest", "%interest".
+   * In saving case: Only for 1. charge type type is "withdrawlfee" or "savings no activity fee" with charge calculation type as "% amount"
+   -->
+            <mat-form-field fxFlex="48%"
+              *ngIf="(chargeForm.controls.chargeAppliesTo.value === 1 && (chargeForm.controls.chargeCalculationType.value === 2 || chargeForm.controls.chargeCalculationType.value === 3 || chargeForm.controls.chargeCalculationType.value === 4)) || (chargeForm.controls.chargeAppliesTo.value === 2 && (chargeForm.controls.chargeTimeType.value === 16 || chargeForm.controls.chargeTimeType.value === 5)&&(chargeForm.controls.chargeCalculationType.value === 2)) || (chargeForm.controls.chargeAppliesTo.value === 4 && (chargeForm.controls.chargeTimeType.value==14 || chargeForm.controls.chargeTimeType.value==15) && chargeForm.controls.chargeCalculationType.value === 2)">
```

Comment:
Loan: Min max should apply for percent of disbursement amount type charge as well

Reply:
<img width="1169" alt="Screenshot 2024-01-03 at 5 34 07 AM" src="https://github.com/openMF/web-app/assets/76156941/81d13e65-b9ff-4363-9117-2b0f3198d303">
In case of Loan entity: 
Min Max should apply for disbursement charge time type only?

Reply:
At backend side min-max is used for disbursement % amount as well.

Reply:
It means disbursement as charge time type with charge calculation type as = %amount only because if it is such it was already covered because I did for all chargeTimeType for %amount in chargeCalculationType?
or we can disbursement as charge time type with any   chargeCalculationType